source/_integrations/agent_dvr.markdown

```diff
@@ -0,0 +1,68 @@
+---
+title: Agent DVR
+description: Access and control Agent DVR from Home Assistant.
+ha_category:
+  - Camera
+ha_config_flow: true
+ha_release: 0.108.6
+ha_iot_class: Local Pull
+ha_codeowners:
+  - '@ispysoftware'
+ha_domain: agent_dvr
+---
+
+[Agent DVR](https://www.ispyconnect.com/download.aspx/) is a free* software DVR solution for windows 10, Mac and Linux. Agent DVR runs as a service or console application and can access and control a huge range of third party cameras with advanced motion detection including YOLO integration for object recognition. The iSpyConnect website provides secured (SSL) remote access without port forwarding needed under a subscription model.
+
+You can add Agent DVR via integrations using the IP address and port of the server e.g.,: `http://192.168.1.3:8090/`.
+
```

Comment:
```suggestion
[Agent DVR](https://www.ispyconnect.com/download.aspx/) is a free* software DVR solution for windows 10, Mac and Linux. Agent DVR runs as a service or console application and can access and control a huge range of third party cameras with advanced motion detection, including YOLO integration for object recognition. The iSpyConnect website provides secured (SSL) remote access without port forwarding needed under a subscription model.

You can add Agent DVR via integrations using the IP address and port of the server, e.g.,: `http://192.168.1.3:8090/`.

```

Reply:
sorry, i can't see what you changed here?

Reply:
added a comma here:
`advanced motion detection, including`

and here:
`of the server, e.g.,:`

Added a new line after this section.